### PR TITLE
add tooltips for easier first time use

### PR DESCRIPTION
**Pull request by rmblau**
I have come across your application and am self-hosting it. Wanted to add some initial tooltips to make it easier for new users to understand what the various icons mean.

### CODE REVIEW

File: beaverhabits/frontend/components.py [hunk 1]
```diff
@@ -35,9 +35,12 @@ def compat_menu(name: str, callback: Callable):
...
 
 
-def menu_icon_button(icon_name: str, click: Optional[Callable] = None) -> Button:
+def menu_icon_button(icon_name: str, click: Optional[Callable] = None, tooltip: Optional[bool]|str = False) -> Button:
     button_props = "flat=true unelevated=true padding=xs backgroup=none"
-    return ui.button(icon=icon_name, color=None, on_click=click).props(button_props)
+    if not tooltip:
+        return ui.button(icon=icon_name, color=None, on_click=click).props(button_props)
+    else:
+         return ui.button(icon=icon_name, color=None, on_click=click).props(button_props).tooltip(tooltip)
```
Comment (DmytroLitvinov) [inline beaverhabits/frontend/components.py:43]: Sorry, I am just subscribed to repo, but not fully aware of stack/libraries etc., but is it possible to do in that project next code:
```
button = ui.button(icon=icon_name, color=None, on_click=click).props(button_props)
if tooltip:
  button = button.tooltip(tooltip)

return button
```

File: beaverhabits/frontend/components.py [hunk 2]
```diff
@@ -35,9 +35,12 @@ def compat_menu(name: str, callback: Callable):
     return ui.menu_item(name, callback).props("dense").classes("items-center")
 
 
-def menu_icon_button(icon_name: str, click: Optional[Callable] = None) -> Button:
+def menu_icon_button(icon_name: str, click: Optional[Callable] = None, tooltip: Optional[bool]|str = False) -> Button:
```
Comment (daya0576) [inline beaverhabits/frontend/components.py:38]: Thanks for contributing to the repo, I just had a chance to review it :p 

The type annotation should be: 
```
tooltip: Optional[str] = None
# or
tooltip: str | None = None
```

File: beaverhabits/frontend/components.py (hunk 1 above)
Comment (daya0576) [reply beaverhabits/frontend/components.py:43]: Yes, it would be great if the code can be reused.

File: beaverhabits/frontend/components.py (hunk 2 above)
Comment (rmblau) [reply beaverhabits/frontend/components.py]: @daya0576  Thank you for the review; I have updated the code with your suggestion and cleaned up the way of returning the button

**Pull request merged**